tests/integration/test_api.py

```diff
@@ -569,3 +569,36 @@ def test_prediction_collection_no_filter(client):
     assert prediction_data[1]["id"] == prediction2.id
     assert prediction_data[1]["type"] == "brand"
     assert prediction_data[1]["value_tag"] == "en:beers"
+
+
+def test_get_unanswered_questions_api(client):
+    result = client.simulate_get("/api/v1/questions/unanswered/")
+
+    assert result.status_code == 200
+    assert result.json == {"questions": [], "status": "no_questions"}
+
+    product1 = ProductInsightFactory(type="category", annotation=0)
```

Comment:
here to test the API the best is to create at least 3 or 5 examples with 2 or 3 different value_tag properties.